tests/pkg/fixture/fixture.go

```diff
@@ -155,6 +155,43 @@ func GetTidbCluster(ns, name, version string) *v1alpha1.TidbCluster {
 	}
 }
 
+func GetTiKVGroup(ns, name, clusterName, version string) *v1alpha1.TiKVGroup {
+	tikvStorageConfig := &v1alpha1.TiKVStorageConfig{
+		// Don't reserve space in e2e tests, see
+		// https://github.com/pingcap/tidb-operator/issues/2509.
+		ReserveSpace: pointer.StringPtr("0MB"),
+	}
+	// We assume all unparsable versions are greater or equal to v4.0.0-beta,
+	// e.g. nightly.
+	if v, err := semver.NewVersion(version); err == nil && v.LessThan(tikvV4Beta) {
+		tikvStorageConfig = nil
+	}
+	tg := &v1alpha1.TiKVGroup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+		},
+		Spec: v1alpha1.TiKVGroupSpec{
+			TiKVSpec: v1alpha1.TiKVSpec{
```

Comment:
do tidb cluster and tikv group share the same `v1alpha1.TiKVSpec`? if it is, we can have a utility function to return `v1alpha1.TiKVSpec`. this avoids duplicated code, and it's easy to update the default tikv spec (for example, adjust the configuration for testing)

Reply:
No, they don't. They only share the same image spec currently.

Reply:
I see.

how about sharing the default TiKV config in GetTiKVGroup/GetTiKVCluster functions? if in the future, we need to update the default testing configuration for TiKV, we only need to update in one place.